utils/unlit/unlit.cabal

```diff
@@ -1,18 +1,14 @@
-cabal-version: 2.4
+cabal-version: 3.0
 Name: unlit
 Version: 0.1
-Copyright: XXX
 License: BSD-3-Clause
-Author: XXX
-Maintainer: XXX
 Synopsis: Literate program filter
-Description: XXX
 Category: Development
 build-type: Simple
-extra-source-files: fs.h
 
 Executable unlit
+    cc-options:       -DFS_NAMESPACE=rts
     Default-Language: Haskell2010
     Main-Is: unlit.c
```

Comment:
@hsyl20 These changes should be upstreamable, I hope? The build-depends and `rts-fs` probably not for now; the rts split upstreaming will likely take a bit more work.

Reply:
Yes removing the XXX should be upstreamable.